app/controllers/landlords_controller.rb

```diff
@@ -0,0 +1,36 @@
+class LandlordsController < ApplicationController
+  def create
+  	if current_user.type == 'Landlord'
+	  	landlord = Landlord.new(landlord_params)
+	  	landlord.save!
+	  	redirect_to landlords_show_url
+	  end
+  end
+
+  def show
+  	if user_signed_in? and current_user.type == 'Landlord'
+  	  @landlord = current_user
+  	  @properties = @landlord.properties
+  	end
+  end
+
+  def update
+  	if user_signed_in?
```

Comment:
the user that updates should be the user whose id matches the id that they provide in the params, as in a random landlord should not be able to update the information of another landlord